Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/architecture/util.cpp

```diff
@@ -0,0 +1,134 @@
+#include "util.hpp"
+
+namespace sne
+{
+
+    bool operator==(const glm::vec3 &v1, const glm::vec3 &v2)
+    {
+        return v1[0] == v2[0] && v1[1] == v2[1] && v1[2] == v2[2];
+    }
+
+    bool operator<(const glm::vec3 &v1, const glm::vec3 &v2) // TO DISCUSS
```

Comment:
I don't think it makes sens to say this. If it is for norms, might be a good idea to create a function with an explicit name instead of this operator.

Reply:
Ok, so is it ok to keep it like that ? 
I currently use it to know if:
(objectif - delta) <= v <= (objectif + delta) 

Sometime, we need to check if a point is at the good place but we do not need a high precision.

Reply:
Ok, well